routes/browse/contributors/render.js

```diff
@@ -1,6 +1,6 @@
 const { page_content_limit, modules, metafields, lazyload, map, DB } = include('config/')
 const load = require('./load/')
-const { array, checklanguage, datastructures } = include('routes/helpers/')
+const { array, checklanguage, datastructures, pagestats } = include('routes/helpers/')
```

Comment:
not sure why we need pagestats on contributor page.

Reply:
team